2017-11-17 22:59:39,200 [Thread-3:3264] [django.request:170] [base:get_response] [WARNING]- Not Found: /images/a1.jpg
2017-11-17 22:59:39,213 [Thread-4:9516] [django.request:170] [base:get_response] [WARNING]- Not Found: /images/a2.jpg
2017-11-17 22:59:39,229 [Thread-5:9704] [django.request:170] [base:get_response] [WARNING]- Not Found: /images/a3.jpg
2017-11-17 22:59:39,242 [Thread-6:11228] [django.request:170] [base:get_response] [WARNING]- Not Found: /images/a4.jpg
2017-11-17 23:23:33,746 [Thread-2:8736] [django.request:256] [base:handle_uncaught_exception] [ERROR]- Internal Server Error: /
Traceback (most recent call last):
  File "C:\Python27\Scripts\scrapytest\lib\site-packages\django\core\handlers\base.py", line 132, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\Users\765150388\PycharmProjects\new_blog\blog\views.py", line 20, in index
    return render(request, 'index.html', locals())
  File "C:\Python27\Scripts\scrapytest\lib\site-packages\django\shortcuts.py", line 67, in render
    template_name, context, request=request, using=using)
  File "C:\Python27\Scripts\scrapytest\lib\site-packages\django\template\loader.py", line 99, in render_to_string
    return template.render(context, request)
  File "C:\Python27\Scripts\scrapytest\lib\site-packages\django\template\backends\django.py", line 74, in render
    return self.template.render(context)
  File "C:\Python27\Scripts\scrapytest\lib\site-packages\django\template\base.py", line 208, in render
    with context.bind_template(self):
  File "c:\python27\Lib\contextlib.py", line 17, in __enter__
    return self.gen.next()
  File "C:\Python27\Scripts\scrapytest\lib\site-packages\django\template\context.py", line 235, in bind_template
    updates.update(processor(self.request))
TypeError: global_settings() takes no arguments (1 given)
2017-11-17 23:25:51,674 [Thread-3:7836] [django.request:170] [base:get_response] [WARNING]- Not Found: /images/a1.jpg
2017-11-17 23:25:51,684 [Thread-4:9048] [django.request:170] [base:get_response] [WARNING]- Not Found: /images/a2.jpg
2017-11-17 23:25:51,694 [Thread-5:9768] [django.request:170] [base:get_response] [WARNING]- Not Found: /images/a3.jpg
2017-11-17 23:25:51,703 [Thread-6:3816] [django.request:170] [base:get_response] [WARNING]- Not Found: /images/a4.jpg
2017-11-18 02:48:59,759 [Thread-8:12056] [django.request:170] [base:get_response] [WARNING]- Not Found: /images/a1.jpg
2017-11-18 02:49:00,101 [Thread-9:9584] [django.request:170] [base:get_response] [WARNING]- Not Found: /images/a2.jpg
2017-11-18 02:49:00,104 [Thread-10:8624] [django.request:170] [base:get_response] [WARNING]- Not Found: /images/a3.jpg
2017-11-18 02:49:00,105 [Thread-11:520] [django.request:170] [base:get_response] [WARNING]- Not Found: /images/a4.jpg
2017-11-18 02:49:01,319 [Thread-25:7452] [django.request:170] [base:get_response] [WARNING]- Not Found: /favicon.ico
2017-11-18 02:49:59,746 [Thread-41:12068] [django.request:170] [base:get_response] [WARNING]- Not Found: /images/a1.jpg
2017-11-18 02:49:59,872 [Thread-42:12364] [django.request:170] [base:get_response] [WARNING]- Not Found: /images/a2.jpg
2017-11-18 02:49:59,997 [Thread-43:10108] [django.request:170] [base:get_response] [WARNING]- Not Found: /images/a3.jpg
2017-11-18 02:50:00,046 [Thread-44:6676] [django.request:170] [base:get_response] [WARNING]- Not Found: /images/a4.jpg
2017-11-18 13:50:21,371 [Thread-8:8584] [django.request:170] [base:get_response] [WARNING]- Not Found: /images/a1.jpg
2017-11-18 13:50:21,421 [Thread-9:8588] [django.request:170] [base:get_response] [WARNING]- Not Found: /images/a2.jpg
2017-11-18 13:50:21,456 [Thread-10:8592] [django.request:170] [base:get_response] [WARNING]- Not Found: /images/a3.jpg
2017-11-18 13:50:21,457 [Thread-11:8596] [django.request:170] [base:get_response] [WARNING]- Not Found: /images/a4.jpg
2017-11-18 13:50:22,753 [Thread-12:8684] [django.request:170] [base:get_response] [WARNING]- Not Found: /favicon.ico
2017-11-18 22:22:00,572 [Thread-3:7196] [django.request:170] [base:get_response] [WARNING]- Not Found: /favicon.ico
2017-11-18 22:23:11,500 [Thread-3:7252] [django.request:170] [base:get_response] [WARNING]- Not Found: /images/a1.jpg
2017-11-18 22:23:11,526 [Thread-4:7496] [django.request:170] [base:get_response] [WARNING]- Not Found: /images/a2.jpg
2017-11-18 22:23:11,556 [Thread-5:7476] [django.request:170] [base:get_response] [WARNING]- Not Found: /images/a3.jpg
2017-11-18 22:23:11,602 [Thread-6:7908] [django.request:170] [base:get_response] [WARNING]- Not Found: /images/a4.jpg
2017-11-18 22:39:09,290 [Thread-3:6792] [django.request:170] [base:get_response] [WARNING]- Not Found: /base.html
2017-11-18 23:53:06,119 [Thread-78:7276] [django.request:170] [base:get_response] [WARNING]- Not Found: /admin/blog/article/images/toolbar/buttons.png/
2017-11-19 14:01:39,525 [Thread-3:7936] [django.request:170] [base:get_response] [WARNING]- Not Found: /favicon.ico
2017-11-19 14:01:51,408 [Thread-6:7884] [django.request:170] [base:get_response] [WARNING]- Not Found: /archive/base.html
2017-11-20 13:33:34,055 [Thread-30:8660] [django.request:170] [base:get_response] [WARNING]- Not Found: /favicon.ico
2017-11-20 13:44:57,706 [Thread-53:5788] [django.request:170] [base:get_response] [WARNING]- Not Found: /article/1
2017-11-20 13:53:18,532 [Thread-60:8204] [django.request:170] [base:get_response] [WARNING]- Not Found: /article/3
2017-11-20 13:53:28,801 [Thread-66:8932] [django.request:170] [base:get_response] [WARNING]- Not Found: /article3
2017-11-20 14:02:16,266 [Thread-85:7264] [django.request:256] [base:handle_uncaught_exception] [ERROR]- Internal Server Error: /article/
Traceback (most recent call last):
  File "C:\Python27\Scripts\scrapytest\lib\site-packages\django\core\handlers\base.py", line 132, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\Users\765150388\PycharmProjects\new_blog\blog\views.py", line 78, in article
    return render(request, 'article.html', locals())
  File "C:\Python27\Scripts\scrapytest\lib\site-packages\django\shortcuts.py", line 67, in render
    template_name, context, request=request, using=using)
  File "C:\Python27\Scripts\scrapytest\lib\site-packages\django\template\loader.py", line 98, in render_to_string
    template = get_template(template_name, using=using)
  File "C:\Python27\Scripts\scrapytest\lib\site-packages\django\template\loader.py", line 35, in get_template
    return engine.get_template(template_name, dirs)
  File "C:\Python27\Scripts\scrapytest\lib\site-packages\django\template\backends\django.py", line 30, in get_template
    return Template(self.engine.get_template(template_name, dirs))
  File "C:\Python27\Scripts\scrapytest\lib\site-packages\django\template\engine.py", line 167, in get_template
    template, origin = self.find_template(template_name, dirs)
  File "C:\Python27\Scripts\scrapytest\lib\site-packages\django\template\engine.py", line 141, in find_template
    source, display_name = loader(name, dirs)
  File "C:\Python27\Scripts\scrapytest\lib\site-packages\django\template\loaders\base.py", line 13, in __call__
    return self.load_template(template_name, template_dirs)
  File "C:\Python27\Scripts\scrapytest\lib\site-packages\django\template\loaders\base.py", line 23, in load_template
    template = Template(source, origin, template_name, self.engine)
  File "C:\Python27\Scripts\scrapytest\lib\site-packages\django\template\base.py", line 190, in __init__
    self.nodelist = engine.compile_string(template_string, origin)
  File "C:\Python27\Scripts\scrapytest\lib\site-packages\django\template\engine.py", line 261, in compile_string
    return parser.parse()
  File "C:\Python27\Scripts\scrapytest\lib\site-packages\django\template\base.py", line 341, in parse
    compiled_result = compile_func(self, token)
  File "C:\Python27\Scripts\scrapytest\lib\site-packages\django\template\loader_tags.py", line 210, in do_extends
    nodelist = parser.parse()
  File "C:\Python27\Scripts\scrapytest\lib\site-packages\django\template\base.py", line 341, in parse
    compiled_result = compile_func(self, token)
  File "C:\Python27\Scripts\scrapytest\lib\site-packages\django\template\defaulttags.py", line 1163, in load
    (taglib, e))
TemplateSyntaxError: 'myfilter' is not a valid tag library: Template library myfilter not found, tried django.templatetags.myfilter,django.contrib.admin.templatetags.myfilter,django.contrib.staticfiles.templatetags.myfilter
2017-11-20 14:02:46,463 [Thread-91:8632] [django.request:256] [base:handle_uncaught_exception] [ERROR]- Internal Server Error: /article/
Traceback (most recent call last):
  File "C:\Python27\Scripts\scrapytest\lib\site-packages\django\core\handlers\base.py", line 132, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\Users\765150388\PycharmProjects\new_blog\blog\views.py", line 78, in article
    return render(request, 'article.html', locals())
  File "C:\Python27\Scripts\scrapytest\lib\site-packages\django\shortcuts.py", line 67, in render
    template_name, context, request=request, using=using)
  File "C:\Python27\Scripts\scrapytest\lib\site-packages\django\template\loader.py", line 98, in render_to_string
    template = get_template(template_name, using=using)
  File "C:\Python27\Scripts\scrapytest\lib\site-packages\django\template\loader.py", line 35, in get_template
    return engine.get_template(template_name, dirs)
  File "C:\Python27\Scripts\scrapytest\lib\site-packages\django\template\backends\django.py", line 30, in get_template
    return Template(self.engine.get_template(template_name, dirs))
  File "C:\Python27\Scripts\scrapytest\lib\site-packages\django\template\engine.py", line 167, in get_template
    template, origin = self.find_template(template_name, dirs)
  File "C:\Python27\Scripts\scrapytest\lib\site-packages\django\template\engine.py", line 141, in find_template
    source, display_name = loader(name, dirs)
  File "C:\Python27\Scripts\scrapytest\lib\site-packages\django\template\loaders\base.py", line 13, in __call__
    return self.load_template(template_name, template_dirs)
  File "C:\Python27\Scripts\scrapytest\lib\site-packages\django\template\loaders\base.py", line 23, in load_template
    template = Template(source, origin, template_name, self.engine)
  File "C:\Python27\Scripts\scrapytest\lib\site-packages\django\template\base.py", line 190, in __init__
    self.nodelist = engine.compile_string(template_string, origin)
  File "C:\Python27\Scripts\scrapytest\lib\site-packages\django\template\engine.py", line 261, in compile_string
    return parser.parse()
  File "C:\Python27\Scripts\scrapytest\lib\site-packages\django\template\base.py", line 341, in parse
    compiled_result = compile_func(self, token)
  File "C:\Python27\Scripts\scrapytest\lib\site-packages\django\template\loader_tags.py", line 210, in do_extends
    nodelist = parser.parse()
  File "C:\Python27\Scripts\scrapytest\lib\site-packages\django\template\base.py", line 341, in parse
    compiled_result = compile_func(self, token)
  File "C:\Python27\Scripts\scrapytest\lib\site-packages\django\template\loader_tags.py", line 184, in do_block
    nodelist = parser.parse(('endblock',))
  File "C:\Python27\Scripts\scrapytest\lib\site-packages\django\template\base.py", line 317, in parse
    filter_expression = self.compile_filter(token.contents)
  File "C:\Python27\Scripts\scrapytest\lib\site-packages\django\template\base.py", line 423, in compile_filter
    return FilterExpression(token, self)
  File "C:\Python27\Scripts\scrapytest\lib\site-packages\django\template\base.py", line 633, in __init__
    filter_func = parser.find_filter(filter_name)
  File "C:\Python27\Scripts\scrapytest\lib\site-packages\django\template\base.py", line 429, in find_filter
    raise TemplateSyntaxError("Invalid filter: '%s'" % filter_name)
TemplateSyntaxError: Invalid filter: 'month_to_upper'
2017-11-20 14:03:47,484 [Thread-2:3500] [django.request:256] [base:handle_uncaught_exception] [ERROR]- Internal Server Error: /
Traceback (most recent call last):
  File "C:\Python27\Scripts\scrapytest\lib\site-packages\django\core\handlers\base.py", line 132, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\Users\765150388\PycharmProjects\new_blog\blog\views.py", line 40, in index
    return render(request, 'index.html', locals())
  File "C:\Python27\Scripts\scrapytest\lib\site-packages\django\shortcuts.py", line 67, in render
    template_name, context, request=request, using=using)
  File "C:\Python27\Scripts\scrapytest\lib\site-packages\django\template\loader.py", line 99, in render_to_string
    return template.render(context, request)
  File "C:\Python27\Scripts\scrapytest\lib\site-packages\django\template\backends\django.py", line 74, in render
    return self.template.render(context)
  File "C:\Python27\Scripts\scrapytest\lib\site-packages\django\template\base.py", line 209, in render
    return self._render(context)
  File "C:\Python27\Scripts\scrapytest\lib\site-packages\django\template\base.py", line 201, in _render
    return self.nodelist.render(context)
  File "C:\Python27\Scripts\scrapytest\lib\site-packages\django\template\base.py", line 903, in render
    bit = self.render_node(node, context)
  File "C:\Python27\Scripts\scrapytest\lib\site-packages\django\template\debug.py", line 79, in render_node
    return node.render(context)
  File "C:\Python27\Scripts\scrapytest\lib\site-packages\django\template\loader_tags.py", line 135, in render
    return compiled_parent._render(context)
  File "C:\Python27\Scripts\scrapytest\lib\site-packages\django\template\base.py", line 201, in _render
    return self.nodelist.render(context)
  File "C:\Python27\Scripts\scrapytest\lib\site-packages\django\template\base.py", line 903, in render
    bit = self.render_node(node, context)
  File "C:\Python27\Scripts\scrapytest\lib\site-packages\django\template\debug.py", line 79, in render_node
    return node.render(context)
  File "C:\Python27\Scripts\scrapytest\lib\site-packages\django\template\defaulttags.py", line 217, in render
    nodelist.append(node.render(context))
  File "C:\Python27\Scripts\scrapytest\lib\site-packages\django\template\defaulttags.py", line 507, in render
    six.reraise(*exc_info)
  File "C:\Python27\Scripts\scrapytest\lib\site-packages\django\template\defaulttags.py", line 493, in render
    url = reverse(view_name, args=args, kwargs=kwargs, current_app=current_app)
  File "C:\Python27\Scripts\scrapytest\lib\site-packages\django\core\urlresolvers.py", line 579, in reverse
    return force_text(iri_to_uri(resolver._reverse_with_prefix(view, prefix, *args, **kwargs)))
  File "C:\Python27\Scripts\scrapytest\lib\site-packages\django\core\urlresolvers.py", line 496, in _reverse_with_prefix
    (lookup_view_s, args, kwargs, len(patterns), patterns))
NoReverseMatch: Reverse for 'article' with arguments '()' and keyword arguments '{}' not found. 1 pattern(s) tried: ['article/(?P<path>.*)$']
2017-11-20 14:28:08,270 [Thread-8:4720] [django.request:256] [base:handle_uncaught_exception] [ERROR]- Internal Server Error: /article/
Traceback (most recent call last):
  File "C:\Python27\Scripts\scrapytest\lib\site-packages\django\core\handlers\base.py", line 132, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\Users\765150388\PycharmProjects\new_blog\blog\views.py", line 79, in article
    return render(request, 'article.html', locals())
  File "C:\Python27\Scripts\scrapytest\lib\site-packages\django\shortcuts.py", line 67, in render
    template_name, context, request=request, using=using)
  File "C:\Python27\Scripts\scrapytest\lib\site-packages\django\template\loader.py", line 99, in render_to_string
    return template.render(context, request)
  File "C:\Python27\Scripts\scrapytest\lib\site-packages\django\template\backends\django.py", line 74, in render
    return self.template.render(context)
  File "C:\Python27\Scripts\scrapytest\lib\site-packages\django\template\base.py", line 209, in render
    return self._render(context)
  File "C:\Python27\Scripts\scrapytest\lib\site-packages\django\template\base.py", line 201, in _render
    return self.nodelist.render(context)
  File "C:\Python27\Scripts\scrapytest\lib\site-packages\django\template\base.py", line 903, in render
    bit = self.render_node(node, context)
  File "C:\Python27\Scripts\scrapytest\lib\site-packages\django\template\debug.py", line 79, in render_node
    return node.render(context)
  File "C:\Python27\Scripts\scrapytest\lib\site-packages\django\template\loader_tags.py", line 135, in render
    return compiled_parent._render(context)
  File "C:\Python27\Scripts\scrapytest\lib\site-packages\django\template\base.py", line 201, in _render
    return self.nodelist.render(context)
  File "C:\Python27\Scripts\scrapytest\lib\site-packages\django\template\base.py", line 903, in render
    bit = self.render_node(node, context)
  File "C:\Python27\Scripts\scrapytest\lib\site-packages\django\template\debug.py", line 79, in render_node
    return node.render(context)
  File "C:\Python27\Scripts\scrapytest\lib\site-packages\django\template\loader_tags.py", line 65, in render
    result = block.nodelist.render(context)
  File "C:\Python27\Scripts\scrapytest\lib\site-packages\django\template\base.py", line 903, in render
    bit = self.render_node(node, context)
  File "C:\Python27\Scripts\scrapytest\lib\site-packages\django\template\debug.py", line 79, in render_node
    return node.render(context)
  File "C:\Python27\Scripts\scrapytest\lib\site-packages\django\template\defaulttags.py", line 329, in render
    return nodelist.render(context)
  File "C:\Python27\Scripts\scrapytest\lib\site-packages\django\template\base.py", line 903, in render
    bit = self.render_node(node, context)
  File "C:\Python27\Scripts\scrapytest\lib\site-packages\django\template\debug.py", line 79, in render_node
    return node.render(context)
  File "C:\Python27\Scripts\scrapytest\lib\site-packages\django\template\defaulttags.py", line 507, in render
    six.reraise(*exc_info)
  File "C:\Python27\Scripts\scrapytest\lib\site-packages\django\template\defaulttags.py", line 493, in render
    url = reverse(view_name, args=args, kwargs=kwargs, current_app=current_app)
  File "C:\Python27\Scripts\scrapytest\lib\site-packages\django\core\urlresolvers.py", line 579, in reverse
    return force_text(iri_to_uri(resolver._reverse_with_prefix(view, prefix, *args, **kwargs)))
  File "C:\Python27\Scripts\scrapytest\lib\site-packages\django\core\urlresolvers.py", line 496, in _reverse_with_prefix
    (lookup_view_s, args, kwargs, len(patterns), patterns))
NoReverseMatch: Reverse for 'logout' with arguments '()' and keyword arguments '{}' not found. 0 pattern(s) tried: []
2017-11-20 15:53:19,392 [Thread-3:9548] [django.request:170] [base:get_response] [WARNING]- Not Found: /favicon.ico
2017-11-20 15:53:27,719 [Thread-8:10180] [django.request:256] [base:handle_uncaught_exception] [ERROR]- Internal Server Error: /article/
Traceback (most recent call last):
  File "C:\Python27\Scripts\scrapytest\lib\site-packages\django\core\handlers\base.py", line 132, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\Users\765150388\PycharmProjects\new_blog\blog\views.py", line 79, in article
    return render(request, 'article.html', locals())
  File "C:\Python27\Scripts\scrapytest\lib\site-packages\django\shortcuts.py", line 67, in render
    template_name, context, request=request, using=using)
  File "C:\Python27\Scripts\scrapytest\lib\site-packages\django\template\loader.py", line 99, in render_to_string
    return template.render(context, request)
  File "C:\Python27\Scripts\scrapytest\lib\site-packages\django\template\backends\django.py", line 74, in render
    return self.template.render(context)
  File "C:\Python27\Scripts\scrapytest\lib\site-packages\django\template\base.py", line 209, in render
    return self._render(context)
  File "C:\Python27\Scripts\scrapytest\lib\site-packages\django\template\base.py", line 201, in _render
    return self.nodelist.render(context)
  File "C:\Python27\Scripts\scrapytest\lib\site-packages\django\template\base.py", line 903, in render
    bit = self.render_node(node, context)
  File "C:\Python27\Scripts\scrapytest\lib\site-packages\django\template\debug.py", line 79, in render_node
    return node.render(context)
  File "C:\Python27\Scripts\scrapytest\lib\site-packages\django\template\loader_tags.py", line 135, in render
    return compiled_parent._render(context)
  File "C:\Python27\Scripts\scrapytest\lib\site-packages\django\template\base.py", line 201, in _render
    return self.nodelist.render(context)
  File "C:\Python27\Scripts\scrapytest\lib\site-packages\django\template\base.py", line 903, in render
    bit = self.render_node(node, context)
  File "C:\Python27\Scripts\scrapytest\lib\site-packages\django\template\debug.py", line 79, in render_node
    return node.render(context)
  File "C:\Python27\Scripts\scrapytest\lib\site-packages\django\template\loader_tags.py", line 65, in render
    result = block.nodelist.render(context)
  File "C:\Python27\Scripts\scrapytest\lib\site-packages\django\template\base.py", line 903, in render
    bit = self.render_node(node, context)
  File "C:\Python27\Scripts\scrapytest\lib\site-packages\django\template\debug.py", line 79, in render_node
    return node.render(context)
  File "C:\Python27\Scripts\scrapytest\lib\site-packages\django\template\defaulttags.py", line 329, in render
    return nodelist.render(context)
  File "C:\Python27\Scripts\scrapytest\lib\site-packages\django\template\base.py", line 903, in render
    bit = self.render_node(node, context)
  File "C:\Python27\Scripts\scrapytest\lib\site-packages\django\template\debug.py", line 79, in render_node
    return node.render(context)
  File "C:\Python27\Scripts\scrapytest\lib\site-packages\django\template\defaulttags.py", line 507, in render
    six.reraise(*exc_info)
  File "C:\Python27\Scripts\scrapytest\lib\site-packages\django\template\defaulttags.py", line 493, in render
    url = reverse(view_name, args=args, kwargs=kwargs, current_app=current_app)
  File "C:\Python27\Scripts\scrapytest\lib\site-packages\django\core\urlresolvers.py", line 579, in reverse
    return force_text(iri_to_uri(resolver._reverse_with_prefix(view, prefix, *args, **kwargs)))
  File "C:\Python27\Scripts\scrapytest\lib\site-packages\django\core\urlresolvers.py", line 496, in _reverse_with_prefix
    (lookup_view_s, args, kwargs, len(patterns), patterns))
NoReverseMatch: Reverse for 'logout' with arguments '()' and keyword arguments '{}' not found. 0 pattern(s) tried: []
2017-11-20 15:55:09,193 [Thread-15:9732] [django.request:256] [base:handle_uncaught_exception] [ERROR]- Internal Server Error: /article/
Traceback (most recent call last):
  File "C:\Python27\Scripts\scrapytest\lib\site-packages\django\core\handlers\base.py", line 132, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\Users\765150388\PycharmProjects\new_blog\blog\views.py", line 79, in article
    return render(request, 'article.html', locals())
  File "C:\Python27\Scripts\scrapytest\lib\site-packages\django\shortcuts.py", line 67, in render
    template_name, context, request=request, using=using)
  File "C:\Python27\Scripts\scrapytest\lib\site-packages\django\template\loader.py", line 99, in render_to_string
    return template.render(context, request)
  File "C:\Python27\Scripts\scrapytest\lib\site-packages\django\template\backends\django.py", line 74, in render
    return self.template.render(context)
  File "C:\Python27\Scripts\scrapytest\lib\site-packages\django\template\base.py", line 209, in render
    return self._render(context)
  File "C:\Python27\Scripts\scrapytest\lib\site-packages\django\template\base.py", line 201, in _render
    return self.nodelist.render(context)
  File "C:\Python27\Scripts\scrapytest\lib\site-packages\django\template\base.py", line 903, in render
    bit = self.render_node(node, context)
  File "C:\Python27\Scripts\scrapytest\lib\site-packages\django\template\debug.py", line 79, in render_node
    return node.render(context)
  File "C:\Python27\Scripts\scrapytest\lib\site-packages\django\template\loader_tags.py", line 135, in render
    return compiled_parent._render(context)
  File "C:\Python27\Scripts\scrapytest\lib\site-packages\django\template\base.py", line 201, in _render
    return self.nodelist.render(context)
  File "C:\Python27\Scripts\scrapytest\lib\site-packages\django\template\base.py", line 903, in render
    bit = self.render_node(node, context)
  File "C:\Python27\Scripts\scrapytest\lib\site-packages\django\template\debug.py", line 79, in render_node
    return node.render(context)
  File "C:\Python27\Scripts\scrapytest\lib\site-packages\django\template\loader_tags.py", line 65, in render
    result = block.nodelist.render(context)
  File "C:\Python27\Scripts\scrapytest\lib\site-packages\django\template\base.py", line 903, in render
    bit = self.render_node(node, context)
  File "C:\Python27\Scripts\scrapytest\lib\site-packages\django\template\debug.py", line 79, in render_node
    return node.render(context)
  File "C:\Python27\Scripts\scrapytest\lib\site-packages\django\template\defaulttags.py", line 329, in render
    return nodelist.render(context)
  File "C:\Python27\Scripts\scrapytest\lib\site-packages\django\template\base.py", line 903, in render
    bit = self.render_node(node, context)
  File "C:\Python27\Scripts\scrapytest\lib\site-packages\django\template\debug.py", line 79, in render_node
    return node.render(context)
  File "C:\Python27\Scripts\scrapytest\lib\site-packages\django\template\defaulttags.py", line 507, in render
    six.reraise(*exc_info)
  File "C:\Python27\Scripts\scrapytest\lib\site-packages\django\template\defaulttags.py", line 493, in render
    url = reverse(view_name, args=args, kwargs=kwargs, current_app=current_app)
  File "C:\Python27\Scripts\scrapytest\lib\site-packages\django\core\urlresolvers.py", line 579, in reverse
    return force_text(iri_to_uri(resolver._reverse_with_prefix(view, prefix, *args, **kwargs)))
  File "C:\Python27\Scripts\scrapytest\lib\site-packages\django\core\urlresolvers.py", line 496, in _reverse_with_prefix
    (lookup_view_s, args, kwargs, len(patterns), patterns))
NoReverseMatch: Reverse for 'logout' with arguments '()' and keyword arguments '{}' not found. 0 pattern(s) tried: []
2017-11-20 16:09:07,204 [Thread-26:8656] [django.request:170] [base:get_response] [WARNING]- Not Found: /base.html
2017-11-20 16:11:29,490 [Thread-2:10152] [django.request:256] [base:handle_uncaught_exception] [ERROR]- Internal Server Error: /article/
Traceback (most recent call last):
  File "C:\Python27\Scripts\scrapytest\lib\site-packages\django\core\handlers\base.py", line 132, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\Users\765150388\PycharmProjects\new_blog\blog\views.py", line 77, in article
    return render(request, 'article.html', locals())
  File "C:\Python27\Scripts\scrapytest\lib\site-packages\django\shortcuts.py", line 67, in render
    template_name, context, request=request, using=using)
  File "C:\Python27\Scripts\scrapytest\lib\site-packages\django\template\loader.py", line 99, in render_to_string
    return template.render(context, request)
  File "C:\Python27\Scripts\scrapytest\lib\site-packages\django\template\backends\django.py", line 74, in render
    return self.template.render(context)
  File "C:\Python27\Scripts\scrapytest\lib\site-packages\django\template\base.py", line 209, in render
    return self._render(context)
  File "C:\Python27\Scripts\scrapytest\lib\site-packages\django\template\base.py", line 201, in _render
    return self.nodelist.render(context)
  File "C:\Python27\Scripts\scrapytest\lib\site-packages\django\template\base.py", line 903, in render
    bit = self.render_node(node, context)
  File "C:\Python27\Scripts\scrapytest\lib\site-packages\django\template\debug.py", line 79, in render_node
    return node.render(context)
  File "C:\Python27\Scripts\scrapytest\lib\site-packages\django\template\loader_tags.py", line 135, in render
    return compiled_parent._render(context)
  File "C:\Python27\Scripts\scrapytest\lib\site-packages\django\template\base.py", line 201, in _render
    return self.nodelist.render(context)
  File "C:\Python27\Scripts\scrapytest\lib\site-packages\django\template\base.py", line 903, in render
    bit = self.render_node(node, context)
  File "C:\Python27\Scripts\scrapytest\lib\site-packages\django\template\debug.py", line 79, in render_node
    return node.render(context)
  File "C:\Python27\Scripts\scrapytest\lib\site-packages\django\template\loader_tags.py", line 65, in render
    result = block.nodelist.render(context)
  File "C:\Python27\Scripts\scrapytest\lib\site-packages\django\template\base.py", line 903, in render
    bit = self.render_node(node, context)
  File "C:\Python27\Scripts\scrapytest\lib\site-packages\django\template\debug.py", line 79, in render_node
    return node.render(context)
  File "C:\Python27\Scripts\scrapytest\lib\site-packages\django\template\defaulttags.py", line 329, in render
    return nodelist.render(context)
  File "C:\Python27\Scripts\scrapytest\lib\site-packages\django\template\base.py", line 903, in render
    bit = self.render_node(node, context)
  File "C:\Python27\Scripts\scrapytest\lib\site-packages\django\template\debug.py", line 79, in render_node
    return node.render(context)
  File "C:\Python27\Scripts\scrapytest\lib\site-packages\django\template\defaulttags.py", line 507, in render
    six.reraise(*exc_info)
  File "C:\Python27\Scripts\scrapytest\lib\site-packages\django\template\defaulttags.py", line 493, in render
    url = reverse(view_name, args=args, kwargs=kwargs, current_app=current_app)
  File "C:\Python27\Scripts\scrapytest\lib\site-packages\django\core\urlresolvers.py", line 579, in reverse
    return force_text(iri_to_uri(resolver._reverse_with_prefix(view, prefix, *args, **kwargs)))
  File "C:\Python27\Scripts\scrapytest\lib\site-packages\django\core\urlresolvers.py", line 496, in _reverse_with_prefix
    (lookup_view_s, args, kwargs, len(patterns), patterns))
NoReverseMatch: Reverse for 'logout' with arguments '()' and keyword arguments '{}' not found. 0 pattern(s) tried: []
2017-11-20 16:56:57,546 [Thread-37:10036] [django.request:170] [base:get_response] [WARNING]- Not Found: /base.html
2017-11-20 17:21:15,426 [Thread-48:8940] [django.request:170] [base:get_response] [WARNING]- Not Found: /css/reglogin.css
2017-11-20 17:21:23,437 [Thread-50:5868] [django.request:170] [base:get_response] [WARNING]- Not Found: /css/reglogin.css
2017-11-20 17:21:30,862 [Thread-52:9936] [django.request:170] [base:get_response] [WARNING]- Not Found: /css/reglogin.css
2017-11-20 17:23:42,724 [Thread-61:7444] [django.request:170] [base:get_response] [WARNING]- Not Found: /css/reglogin.css
2017-11-20 17:23:43,828 [Thread-63:2292] [django.request:170] [base:get_response] [WARNING]- Not Found: /css/reglogin.css
2017-11-20 17:24:34,726 [Thread-10:7736] [django.request:170] [base:get_response] [WARNING]- Not Found: /css/reglogin.css
2017-11-20 17:28:55,805 [Thread-12:5104] [django.request:170] [base:get_response] [WARNING]- Not Found: /css/reglogin.css
2017-11-20 18:12:52,944 [Thread-19:6960] [django.request:170] [base:get_response] [WARNING]- Not Found: /favicon.ico
2017-11-20 21:33:38,382 [Thread-29:8304] [django.request:170] [base:get_response] [WARNING]- Not Found: /favicon.ico
2017-11-20 21:40:39,740 [Thread-14:7496] [django.request:170] [base:get_response] [WARNING]- Not Found: /article/base.html
2017-11-20 22:02:25,005 [Thread-9:11240] [django.request:170] [base:get_response] [WARNING]- Not Found: /article/base.html
2017-11-20 22:31:15,092 [Thread-2:10104] [django.request:170] [base:get_response] [WARNING]- Not Found: /base.html
2017-11-20 23:19:47,270 [Thread-8:10728] [django.request:170] [base:get_response] [WARNING]- Not Found: /base.html
